### PR TITLE
feat(ButtonLink): change chevron icon in Vivo-new skin

### PR DESCRIPTION
**Pull request by atabel**
WEB-1809

### CODE REVIEW

File: src/button.tsx [hunk 1]
```diff
@@ -81,6 +82,22 @@ const renderButtonElement = ({
...
 
 const ButtonLinkChevron: React.FC = () => {
+    const {skinName} = useTheme();
+
+    // vivo new skin has a different chevron
+    if (skinName === VIVO_NEW_SKIN) {
+        return (
+            // w = 4px / 14px = 0.2857em
+            // h = 6px / 14px = 0.4286em
+            <svg width="0.2857em" height="0.4286em" viewBox="0 0 4 6" fill="none">
```
Comment (atabel) [inline src/button.tsx:92]: please, review this sizes. I copied the svg from figma with a size of 4x6px and translated it to `em` (button links have a font size of 14)

Comment (yceballost) [reply src/button.tsx:92]: I think you didn't copy the correct layer

The container should be 0.5em like in the line 102
image

**Review by yceballost — CHANGES_REQUESTED** [review]

File: src/button.tsx (hunk 1 above)
Comment (pladaria) [reply src/button.tsx:92]: I guess that's why the chevron looks weird:
image

Comment (atabel) [reply src/button.tsx:92]: copying from figma is a nightmare :(

Comment (atabel) [reply src/button.tsx]: should be fixed now

**Review by marcoskolodny — APPROVED** [review]

**Review by yceballost — APPROVED** [review]
approved by Vivo design team
image

**Review by pladaria — APPROVED** [review]

**Pull request merged**